include/picongpu/particles/ionization/byField/BSI/AlgorithmBSI.hpp

```diff
@@ -71,22 +71,21 @@ namespace ionization
             {
                 uint32_t const cs = pmacc::math::float2int_rd(chargeState);
                 /* ionization potential in atomic units */
-                float_X const iEnergy = typename GetIonizationEnergies<ParticleType>::type{ }[cs];
+                float_X const ionizationEnergy = typename GetIonizationEnergies<ParticleType>::type{ }[cs];
```

Comment:
Why'd you rename `iEnergy` to `ionizationEnergy` in only one of the `Algorithm<...>` files but not in all?
Please make it consistent or revert the one in this file :)